lib/tasks_private/spec_helper.rake

```diff
@@ -65,16 +72,26 @@ class PopulateTower
       c.basic_auth(id, password)
     end
 
-    # Get API version.
+    @tower_data = {}
+
     uri = '/api/v1/config'
-    obj = JSON.parse(@conn.get(uri).body)
-    @version = Gem::Version.new(obj['version'])
+    config = get_obj(uri)
+    @version = Gem::Version.new(config['version'])
+    @tower_data[:config] = { :version => config['version'] }
+
+    uri = '/api/v1/me'
+    me = get_obj(uri)
+    @tower_data[:user] = { :id => me['results'].first['id'] }
   end
 
   def v3_2?
     @version >= Gem::Version.new('3.2')
   end
 
+  def save
```

Comment:
This object is `PopulateTower` and so a method named as `save` would be ambiguous as to where it's saving to. How about rename it explicitly as `to_file`, `data_to_file` or `dump`?